engine/crates/composition/tests/composition/cost_basic/api.graphql

```diff
@@ -0,0 +1,19 @@
+enum Blah @cost(weight: 5) {
+    BLAH
+}
+
+type Query @cost(weight: 1) {
+    foo(name: String @cost(weight: 100)): String
+    id: ID! @cost(weight: 2)
```

Comment:
I'm not familiar with the directive yet so I don't know: is the directive supposed to be in the public API exposed by the router or is it an internal federation directive? (like `@key` & co)

Reply:
The part I don't understand is that in the implementation below it's marked as non public, but it's in this snapshot.

Reply:
Ah I guess what's happening is that it comes back as `Directive::Other` because `@cost` isn't handled in `from_sdl()` in the federated-graph crate.

Reply:
Yeah, good spot.  It was not meant to be in here, it's just very easy to not notice how many spapshots you're updating when you use UPDATE_EXPECT.